web-client/src/ducks/requests/types.ts

```diff
@@ -11,72 +11,70 @@ export const CHANGE_MODAL = syncType('CHANGE_MODAL');
 
```

Comment:
- Same as previous - renaming and separating out FIND.  
- Added type for PostWithOffersAndTimeline - holds the offers and timelines for a single post